src/woo_publications/publications/api/serializers.py

```diff
@@ -212,3 +218,18 @@ def validate_publicatiestatus(
                 )
 
         return value
+
+    @transaction.atomic
+    def update(self, instance, validated_data):
+        publication = super().update(instance, validated_data)
```

Comment:
```suggestion
    @transaction.atomic
    def update(self, instance, validated_data):
        assert instance.publicatiestatus != PublicationStatusOptions.revoked
        publication = super().update(instance, validated_data)
```
Our design and assumption now assumes you cannot update a revoked record anymore, we can express this in code with an assertion that serves as documentation which will cause crashes in development if/when behaviour changes (and then is the time to revisit/check our assumptions again).